csharp/Platform.Collections/Arrays/ArrayPool[T].cs

```diff
@@ -44,12 +70,31 @@ public Disposable<T[]> Resize(Disposable<T[]> source, long size)
             return destination;
         }
 
+        /// <summary>
+        /// <para>Clears the pool.</para>
+        /// <para>Очищает пул.</para>
+        /// </summary>
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         public virtual void Clear() => _pool.Clear();
 
+        /// <summary>
+        /// <para>Retrieves an array with the specified size from the array pool.</para>
+        /// <para>Извлекает из пула массивов массив с указанным размером.</para>
+        /// </summary>
+        /// <typeparam name="T"><para>The array elements type.</para><para>Тип элементов массива.</para></typeparam>
```

Comment:
This method does not have type parameters, the class does. This comment should be moved under the missing class' summary.